packages/addons/tailwindcss/index.ts

```diff
@@ -120,6 +120,15 @@ export default defineAddon({
 			});
 		}
 
+		sv.file('.vscode/settings.json', (content) => {
```

Comment:
I'm not sure how should we deal with all different `ide` support.

A/ Should we have a list of "offical support"?
B/ Should we add a question? Or just try to smart detect (like `TERM_PROGRAM=vscode`)

@manuel3108 @benmccann any insights?

Reply:
All right 👍
We can move with https://github.com/sveltejs/cli/issues/626 as well later then